rpc.go

```diff
@@ -7,6 +7,8 @@ import (
 
 	v1 "github.com/roadrunner-server/api/v4/build/applogger/v1"
 
+	"encoding/json"
```

Comment:
Please, use goccy/go-json as in other plugins, it is faster than a std json, until we're waiting for the std json v2.

Reply:
fixed